core/script.runtime/src/kotlin/script/dependencies/resolvers.kt

```diff
@@ -32,6 +50,7 @@ interface DependenciesResolver : @Suppress("DEPRECATION") ScriptDependenciesReso
 
     sealed class ResolveResult {
```

Comment:
I believe that we better avoid additional documentation here. Maybe something like this:
```
data class ResolutionResult(
    val dependencies: ScriptDependencies,
    val diagnostics: List<ScriptDiagnostic>
    val isSuccess get() = diagnostics.none { it.severity == ScriptDiagnostic.Severity.ERROR }
)
```